src/content/reference/web-interface/_index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Web Interface Reference"
+description = "Documentation on our web interface, a visual way to create and delete clusters, create key pairs and more."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# Giant Swarm's Web Interface
```

Comment:
```suggestion
# Web User Interface
```

I think the fact that this is for Giant Swarm is self-evident based on the context.

Reply:
I'd prefer `The Web Interface` or `Web Interface` here instead.

See ongoing discussion here: https://github.com/giantswarm/giantswarm/issues/6122